design/features/routing/version-1.md

```diff
@@ -28,57 +28,57 @@ ___
+- Form creator **must** select a next question for each radio option **if** one radio option in a list has had a next question set  
+- Form creator can set next (single) question as skippable if radio option X is chosen  
 
 
 ## Key decisions
 
-- we are only going to allow routing to be based on a radio button answer type
-- there can only be one route for a question
-  - a question can be routed to and then routed from
-  - but cannot have multiple routes from
-- we will not be considering branching journeys as part of this work
-- we will prioritise errors for breaking changes, such as
-  - the answer to base the route on is changed or deleted
+- We're only going to allow routing to be set up from a radio button answer type 
+- Only one route can be set up for each question
+  - A question can be routed to and then routed from
+  - A question cannot have multiple routes from it
+- We will not consider branching journeys as part of this work
+- We'll prioritise errors for breaking changes, such as:
+  - the answer the route is based on is changed or deleted
   - the question the route takes the person to is deleted
-  - the question the route takes the person to is now above the question the route should start
-- we will also give an error for when a question the route takes the person to is now directly below the question the route starts from 
-  - this could make our database complicated and have an impact on future iterations
-  - this is mostly house keeping
-- we will not allow a form to be made “Live” without the breaking change errors being resolved
-  - this is to reduce poor forms, and infinite loops within forms
-- we will not allow a creator to change the question the route is based on when editing a route
-  - the user should delete a route in this situation, as they will need to select all the different parts of the route
-  - this is something we might revisit, but makes development easier for now
-- moving a page that has a route attached (starting question) will also move the route
-- if the route start question is deleted, the route will also be deleted
-- if the route start question answer type is changed from radio options, the route will be deleted. This includes
-  - answer type being changed
-  - option for “People can only select on option” is unselected on the “Create a list of options” screen, making the question a checkbox and not a radio type question
+  - the question the route takes the person to is now above the question the route should start from
+- We'll also show an error when a question the route takes the person to is now directly below the question the route starts from 
+  - This could make our database complicated and have an impact on future iterations
```

Comment:
Not quite sure what this bit means (or what the 'This' refers to)

Reply:
The ‘this’ refers to the situation described in the above bullet point. It’s an extra level of detail to explain why we consider it to be an essential error to cover as part of the initial breaking changes error work we did

Reply:
I'm still not sure what it is that could make our database complicated exactly? (And what the relevance of this is here.) 

So are we saying that if someone moves a route's destination question so that it sits directly below the route's start question it not only makes the route unnecessary but also complicates our database somehow?

Could we be a bit more explicit about how it would 'complicate' the database?

Maybe say something like 'Besides making the route redundant, this could cause [bit more detail about how it would affect the database]'?

Reply:
Removed